meetings/2023-07/july-11.md

```diff
@@ -523,9 +580,16 @@ RPR: The discussion has concluded and we have welcomed JHD and CDA back in. So,
 CDA: All right. Fabulous.
 (applause)
 
+### Summary
+
+An update on TG3 has been given:
+
+- https://www.ecma-international.org/task-groups/tc39-tg3/
+- [slides](https://drive.google.com/file/d/1MPHGzy4aH_vRnduuuuUucP7xq_clcrK2/)
+
 ### Conclusion
 
-Consensus on JHD and CDA as convenors of TG3.
+TC39 noted and approved the update and had Consensus on JHD (Jordan Harband) and CDA (Chris de Almeida) as new convenors of TG3. TC39 wished them successful work.
```

Comment:
```suggestion
TC39 noted and approved the update and had Consensus on JHD and CDA as new convenors of TG3. TC39 wished them successful work.
```
full names should only appear at the table in the top of the document; we use acronyms everywhere else.